schemas/stsci.edu/asdf/0.1.0/core/column.yaml

```diff
@@ -0,0 +1,47 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/0.1.0/core/column"
+tag: "tag:stsci.edu:asdf/0.1.0/core/column"
+
+title: >
+  A column in a table.
+
+description: |
+  Each column contains a name and an array of data, and an optional description
+  and unit.
+
+type: object
+properties:
+  name:
+    description: The name of the column
+    type: string
+    pattern: "[A-Za-z_][A-Za-z0-9_]*"
```

Comment:
Okay, for now we can assume "YAGNI".  I admit for most astronomy data it seems like it would be a rarity.  But since ASDF is aiming to be useful beyond astronomy the lack of international support for column names makes me sad.